diff --git a/front-end/src/pages/Login.jsx b/front-end/src/pages/Login.jsx
index 15b16a2..02787ef 100644
--- a/front-end/src/pages/Login.jsx
+++ b/front-end/src/pages/Login.jsx
@@ -1,10 +1,55 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { useContext } from 'react';
+import { Link } from 'react-router-dom';
+import Button from '../components/Button';
+import Input from '../components/Input';
+import GlobalContext from '../context/GlobalContext';
+import { loginValidate } from '../utils/validation';
 
 function Login() {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [disabled, setDisabled] = useState(true);
+  const { loginUser, error } = useContext(GlobalContext);
+
+  useEffect(() => {
+    const result = loginValidate(email, password);
+    setDisabled(result);
+  }, [email, password]);
+
   return (
-    <div>
-      <h1>Login</h1>
-    </div>
+    <main className="containerLoginRegister">
+      <h1>Login 2</h1>
+      <span>Email</span>
+      <Input
+        testId="common_login__input-email"
+        type="email"
+        name="email"
+        value={ email }
+        handleChange={ (e) => setEmail(e.target.value) }
+      />
+      <span>Password</span>
+      <Input
+        testId="common_login__input-password"
+        type="password"
+        name="password"
+        value={ password }
+        handleChange={ (e) => setPassword(e.target.value) }
+      />
+      <Button
+        testId="common_login__button-login"
+        text="LOGIN"
+        disabled={ disabled }
+        handleClick={ () => loginUser({ email, password })}
+      />
+      <Link to="/register">
+        <Button
+          testId="common_login__button-register"
+          text="Ainda não tenho conta"
+        />
+      </Link>
+      { error && <span data-testid="common_login__element-invalid-email">{error.message}</span> }
+    </main>
   );
 }
 
diff --git a/front-end/src/pages/Register.jsx b/front-end/src/pages/Register.jsx
index 7ac7570..a917e94 100644
--- a/front-end/src/pages/Register.jsx
+++ b/front-end/src/pages/Register.jsx
@@ -1,8 +1,53 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import Button from '../components/Button';
+import Input from '../components/Input';
+import { registerValidate } from '../utils/validation';
 
 function Register() {
+  const [name, setName] = useState('');
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [disabled, setDisabled] = useState(true);
+
+  useEffect(() => {
+    const result = registerValidate(name, email, password);
+    setDisabled(result);
+  }, [name, email, password]);
+
   return (
-    <h1>Register</h1>
+    <main className="containerLoginRegister">
+      <h1>Register</h1>
+      <span>Nome</span>
+      <Input
+        testId="common_register__input-name"
+        type="text"
+        name="name"
+        value={ name }
+        handleChange={ (e) => setName(e.target.value) }
+      />
+      <span>Email</span>
+      <Input
+        testId="common_register__input-email"
+        type="email"
+        name="email"
+        value={ email }
+        handleChange={ (e) => setEmail(e.target.value) }
+      />
+      <span>Senha</span>
+      <Input
+        testId="common_register__input-password"
+        type="password"
+        name="password"
+        value={ password }
+        handleChange={ (e) => setPassword(e.target.value) }
+      />
+      <Button
+        testId="common_register__button-register"
+        text="CADASTRAR"
+        disabled={ disabled }
+      />
+      <span data-testid="common_register__element-invalid_register"> </span>
+    </main>
   );
 }
 
